### DISCUSSION TITLE
Problems with r-universe

### DISCUSSION THREAD

**Discussion by mbojan** [wlandau/instantiate#11] [General]
I'm testdriving **instantiate** in https://github.com/coalesce-lab/stansum. Everything works seamlessly apart from r-universe building, c.f. coalesce-lab/stansum#2. Perhaps you have ideas @wlandau ?

**Comment by wlandau** [comment]
From https://github.com/r-universe/coalesce-lab/actions/runs/6441626642/job/17549931825#step:3:301, it looks like `cmdstanr` needs to be installed. Unfortunately this is tricky to do automatically from `install.packages("instantiate")` because `cmdstanr` is not on CRAN. It may help to set up a personal `r-universe`: https://github.com/r-universe-org/help#how-to-setup-your-personal-universe

**Reply by mbojan** [reply]
Thanks @wlandau . I did setup private r-universe at https://github.com/coalesce-lab/universe and https://coalesce-lab.r-universe.dev/builds. Do you know ways to customize the build process?

**Reply by wlandau** [reply]
Sorry, I don't know of a way offhand. Might be useful to check out https://github.com/r-universe-org/help
